templates/project/Makefile

```diff
@@ -15,12 +15,12 @@ upload:
 		--erase-parts nvs \
 		--monitor \
 		--partition-table partitions.csv \
-		--baud ${BAUDRATE} \
+		--baud 115200 \
```

Comment:
didn't want to change the`BAUDRATE` itself on L5?

Reply:
no better to have higher baudrate when flashing

Reply:
but that shouldn't have changed :P